CHANGES.rst

```diff
@@ -6,6 +6,11 @@ ami
 
 - Reorganized step documentation [#4697]
 
+assign_wcs
+---
```

Comment:
Underline needs to be same length as section title in line 9

Reply:
Fixed